sys/src/libacpi/harvey.c

```diff
@@ -113,7 +116,7 @@ ins(uint16_t addr)
 	uint64_t off = addr;
 	uint16_t w;
 	if (pread(acpiio, &w, 2, off) < 2)
-		print("ins(0x%x): %r\n", addr);
+		if(debug) DBG("ins(0x%x): %r\n", addr);
```

Comment:
Might be better to wrap if(debug) in the DBG macro.

Reply:
Ron left int debug at the begining for flag use. For gdb or compiling time. I tried to be less intrusive too here. Acpi needs a general clean and a homogeneus way for switching flags for debugging. No more macros. That file is full of "if(debug)".

Reply:
We should have to redo this part with a label or so.